model-optimizer/extensions/ops/MatMul.py

```diff
@@ -162,10 +162,6 @@ def infer(node: Node):
         Raises on any shape inconsistency
         """
         name = node.soft_get('name', str(node.id))
-        connected_in_ports = {idx: port for idx, port in node.in_ports().items() if not port.disconnected()}
```

Comment:
Please do not remove this lines. They are valid. If PyTorch has MatMul that has different semantic -- we should normalize it separately on the front phase

Reply:
It's about `DecomposeBias` logic. I need to create a fully connected layer with bias.

Reply:
Reverted, thanks!